src/screens/RecipesList/useRecipesListController.ts

```diff
@@ -32,26 +32,26 @@ export const useRecipeListController = () => {
   useEffect(() => {
     setLoading(true);
 
-    Promise.all([getRecipes(searchOptions), getCardType()])
+    Promise.all([getRecipes({ searchTerm: searchOptions.searchTerm, ...searchOptions.pendingOptions }), getCardType()])
```

Comment:
I believe it is worth having a dedicated function/memorized object for storing the merged options